templates/articles/article_index_page.html

```diff
@@ -17,21 +17,17 @@ <h2 class="sr-only">{% trans "Each collection insight takes an in-depth look int
     {% if page.featured_article %}
     <div class="container">
         <div class="featured-article">
-            {% if page.featured_article.hero_image %}
-                <a href="{% pageurl page.featured_article %}" class="featured-article__image-link" tabindex="-1" aria-hidden="true" data-component-name="Featured Article: {{ page.featured_article.title }}" data-link-type="Image" {% if page.featured_article.is_newly_published %}data-label="New"{%endif%}>
+            {% if page.featured_article.specific.teaser_image %}
+                <a href="{% pageurl page.featured_article %}" class="featured-article__image-link" tabindex="-1" aria-hidden="true" data-component-name="Featured Article: {{ page.featured_article.title }}" data-link-type="Image" {% if page.featured_article.specific.is_newly_published %}data-label="New"{%endif%}>
```

Comment:
1. This image change - from `hero image` to `teaser image` does not look like it is part of the requirements.
2. The `teaser image` is non mandatory, do you want to have a _fall-back_ to `hero image` which is mandatory, otherwise when teaser image is not entered, no image will get displayed?
3. Why is `.specific` used with `teaser image`, previous `hero image` did not use it?

Reply:
Hi Allan, the previous one was using hero image in error. I noticed this and changed it here, as it was mixing two different images... Checking to see if it had a hero image but then using the teaser image.

Specific has to be used because of the nature of the `PageChooserPanel`. It doesn't get you anything other than the page title, so specific has to be used to get the page images, teaser text, etc

Reply:
> I noticed this and changed it here

Can this be included somewhere in the requirements to say it also fixes this issue ?

>  Checking to see if it had a hero image but then using the teaser image.

see point 2) above for this

Reply:
@jamesbiggs thanks the chat and clarifications - Record Article page does not have hero image; both Record Article page and Article page have Teaser image and we want to use teaser image.